skimage/filters/_frangi.py

```diff
@@ -0,0 +1,185 @@
+import numpy as np
+
+
+__all__ = ['frangi_filter', 'hessian_filter']
+
+
+def frangi_filter(image, scale=(1, 10), scale_ratio=2, beta1=0.5, beta2=15,
+                  black_white=True):
+    """Returns image filtered with frangi filter.
+
+    Calculates the eigenvectors of the Hessian to compute the likeliness of
+    an image region to vessels, according to the method described in _[1].
+
+    Parameters
+    ----------
+    image : (N, M) ndarray
+        Array with input image data.
+    scale : tuple, optional
+        The range of sigmas used, default [1 10].
+    scale_ratio : int, optional
+        Step size between sigmas, default 2.
+    beta1 : float, optional
+        Frangi correction constant, default 0.5.
+    beta2 : float, optional
+        Frangi correction constant, default 15.
+    black_white : boolean, optional
+        Detect black ridges (default) set to true, for
+        white ridges set to false.
```

Comment:
This is a pretty terrible parameter name... I would suggest `black_ridges`, set to True it detects black ridges, set to false it detects white ones.